### ISSUE TITLE
incorrect time units in pg_ctl timeout value

### ISSUE THREAD

**Issue by YanChii** [dalibo/PAF#54]
Hi,
there is an inconsistency in interpreting timeout values. The cluster settings require value in seconds but the agent itself receives timeout value in miliseconds.

With the following setup

```
root@cluster1:~# pcs resource show postgres-ha
 Master: postgres-ha
  Meta Attrs: master-node-max=1 master-max=1 clone-max=2 notify=True clone-node-max=1 target-role=Stopped
  Resource: postgres (class=ocf provider=heartbeat type=pgsqlms)
   Attributes: pgdata=/var/lib/pgsql/9.5/data pgport=5432 bindir=/usr/pgsql-9.5/bin
   Operations: start interval=0s timeout=60s (postgres-start-interval-0s)
               stop interval=0s timeout=60s (postgres-stop-interval-0s)
               promote interval=0s timeout=30s (postgres-promote-interval-0s)
               demote interval=0s timeout=120s (postgres-demote-interval-0s)
               notify interval=0s timeout=60s (postgres-notify-interval-0s)
               monitor interval=15s role=Master timeout=10s (postgres-monitor-interval-15s)
               monitor interval=16s role=Slave timeout=10s (postgres-monitor-interval-16s)
```
E.g. for the start command, the OCF_RESKEY_CRM_meta_timeout variable is 60000.

And the resulting start command looks like:
```DEBUG: _runas: launching as "postgres" command "/usr/pgsql-9.5/bin/pg_ctl --pgdata /var/lib/pgsql/9.5/data -w --timeout 60060 start"```

Jan

**Comment by ioguix** [comment]
Thank you for the bug report. I'll fix this as soon as possible.

The fix will be in next 2.1rc2.

**Comment by ioguix** [comment]
Thank you for your patch!
